src/workerd/api/actor-alarms-test.js

```diff
@@ -11,8 +11,9 @@ export class DurableObjectExample {
 
   async waitForAlarm(scheduledTime) {
     let self = this;
-    let prom = new Promise((resolve) => {
+    let prom = new Promise((resolve, reject) => {
       self.resolve = resolve;
+      self.reject = reject;
     });
```

Comment:
nit: `const { promise, resolve, reject } = Promise.withResolvers();` is probably a bit cleaner here.

```js
const { promise, resolve, reject } = Promise.withResolvers();
self.resolve = resolve;
self.reject = reject;

// ...

try {
  await promise;
  /// ...
```

Reply:
Oh, nice... I'd forgotten that this had been added.